packages/cli/package.json

```diff
@@ -53,6 +53,7 @@
   },
   "dependencies": {
     "@types/degit": "^2.8.3",
+    "axios": "^1.4.0",
```

Comment:
The project is on node v18 already; can you use native `fetch` instead?

Reply:
Even though the project is using fetch, this'd be packaged in a CLI. I've used node-fetch as per https://github.com/triggerdotdev/trigger.dev/pull/174#discussion_r1271930801